test/unit-test/main.tf

```diff
@@ -1,7 +1,7 @@
 
 module "module_test" {
   source                 = "../../"
-  github_repository      = "ministryofjustice/modernisation-platform-environments:*"
+  github_repositories    = ["ministryofjustice/modernisation-platform-environments:*", "ministryofjustice/modernisation-platform-ami-builds:*"]
```

Comment:
👍 good to add this to the tests, is there a way we can verify in the test that both repos have been added?